server/prisma/schema.prisma

```diff
@@ -0,0 +1,28 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model ChunkData {
+  hash        Bytes         @id
+  version     Int
+  data        Bytes
```

Comment:
missing world/chunkX/chunkZ/timestamp, will be in next commit I presume but adding this comment so I remember